subsys/mpsl/mpsl_init.c

```diff
@@ -171,6 +172,9 @@ static int mpsl_lib_init(const struct device *dev)
 		return err;
 	}
 
+mpsl_fem_device_config_254_apply_set(
```

Comment:
Nitpick, lost the indentation there
```suggestion
	mpsl_fem_device_config_254_apply_set(
```

Reply:
Fixed